requirements.txt

```diff
@@ -7,4 +7,5 @@ torch-harmonics
 matplotlib
 opt-einsum
 h5py
-zarr
\ No newline at end of file
+zarr
+torch
```

Comment:
@m4e7 the reason I hadn't included torch in the requirements is that typically users don't want to install it with pip (conda, Docker etc typically are better to handle CUDA installs) and this can cause issues particularly with `pip -U` where it may override existing install with wrong version.

Reply:
Aha, got it. I didn't realize that was a common user workflow. I've removed that line from `requirements.txt`